src/pluto/pluto.py

```diff
@@ -422,111 +325,325 @@ def generate_graph(v1, v2):
-         For example, the mismatch graph for finance may show that over 90% of lots get an updated assessment when the tentative roll is released.
-         The aggregate graph may show that the aggregated sum of assessments increased by 5% compared with the previous version.\n
-         Totals for assessland, assesstot, and exempttot should only change in February and June.\n
-         Special Notes:\n
-         1. Y-axis represents percent change over the previous version. \n
-         2. Totals for assessland, assesstot, and exempttot should only change in February and June.\n
-         3. Pay attention to any large changes to residential units (unitsres).
-        """
         )
 
-    def create_expected(df, v1, v2):
-
-        exp = df[df["v"].isin([v1, v2])]
-
-        exp_records = exp.to_dict("records")
-        v1_exp = [i["expected"] for i in exp_records if i["v"] == v1][0]
-        v2_exp = [i["expected"] for i in exp_records if i["v"] == v2][0]
-        for field in [
-            "zonedist1",
-            "zonedist2",
-            "zonedist3",
-            "zonedist4",
-            "overlay1",
-            "overlay2",
-            "spdist1",
-            "spdist2",
-            "spdist3",
-            "ext",
-            "proxcode",
-            "irrlotcode",
-            "lottype",
-            "bsmtcode",
-            "bldgclasslanduse",
-        ]:
-            val1 = [i["values"] for i in v1_exp if i["field"] == field][0]
-            val2 = [i["values"] for i in v2_exp if i["field"] == field][0]
-            in1not2 = [i for i in val1 if i not in val2]
-            in2not1 = [i for i in val2 if i not in val1]
-            if len(in1not2) == 0 and len(in2not1) == 0:
-                pass
-            else:
-                st.markdown(f"### Expected value difference for {field}")
-                if len(in1not2) != 0:
-                    st.markdown(f"* in {v1} but not in {v2}:")
-                    st.write(in1not2)
-                if len(in2not1) != 0:
-                    st.markdown(f"* in {v2} but not in {v1}:")
-                    st.write(in2not1)
-
-    create_mismatch(data["df_mismatch"], v1, v2, v3, condo, mapped)
-
-    create_null(data["df_null"], v1, v2, v3, condo, mapped)
-
-    create_aggregate(data["df_aggregate"], v1, v2, v3, condo, mapped)
-
-    st.header("Source Data Versions")
-    code = st.checkbox("code")
-    if code:
-        st.code(data["version_text"])
-    else:
-        st.markdown(data["version_text"])
-
-    # EXPECTED VALUE
-    st.header("Expected Value Comparison")
-    st.write(
-        "if nothing showed up, then it means there aren't any expected value change"
-    )
+        applied_corrections = data['pluto_corrections_applied']
+        not_applied_corrections = data['pluto_corrections_not_applied']
+
+        if applied_corrections is None or not_applied_corrections is None:
+            st.info("There are no available corrections reports for this branch.")
```

Comment:
Maybe we could include a sentence like "this is probably due to a problem on the backend" or something like that?